jhu/delphi_jhu/geo.py

```diff
@@ -30,22 +31,22 @@ def geo_map(df: pd.DataFrame, geo_res: str):
     df = df.copy()
     if geo_res not in VALID_GEO_RES:
         raise ValueError(f"geo_res must be one of {VALID_GEO_RES}")
-
+    unassigned_counties = df[df["fips"].str.endswith("000")].copy()
+    df = df[~df["fips"].str.endswith("000")].copy()
     gmpr = GeoMapper()
+    df = gmpr.add_population_column(df, "fips")
     if geo_res == "county":
-        df.rename(columns={'fips': 'geo_id'}, inplace=True)
-    elif geo_res == "state":
-        df = df.set_index("fips")
-        # Zero out the state FIPS population to avoid double counting.
-        state_fips_codes = {str(x).zfill(2) + "000" for x in range(1,73)}
-        subset_state_fips_codes = set(df.index.values) & state_fips_codes
-        df.loc[subset_state_fips_codes, "population"] = 0
-        df = df.reset_index()
-        df = gmpr.replace_geocode(df, "fips", "state_id", new_col="geo_id", date_col="timestamp")
+        if not sensor.endswith("_prop"):
```

Comment:
Ah, I think this misses the `incidence` signal, which unfortunately is only named `incidence_prop` when it's exported (see `SENSOR_NAME_MAP` in `run.py`).

Reply:
The export csv testing probably does not catch it because line 112 in `run.py` removes all nan values. That line is a little too trigger happy imo.

Reply:
good catch, thanks